default.nix

```diff
@@ -20,6 +20,10 @@ let ic-ref-run =
       cp ${ic-hs-pkgs.ic-hs}/bin/ic-ref-run $out/bin
   ''; in
 
+let
+  nixos-unstable = import <nixos-unstable> {};
+in
```

Comment:
Thanks for this suggestion. I will try this. Is there also a way to add nix packages from the unstable channel?

Reply:
I would need the latest `emscripten` which has a lot of other dependencies...

Reply:
Yes, add `nixpkgs-unstable`, tracking `master` rather than `release-22.05` to sources.json with `niv`, next to the existing `nixpkgs`, and then use
```suggestion
let
  nixos-unstable = import nixpkgs.sources.ic-hs {};
in
```
here. No need to deal with channels per se (which are more suitable for user’s environments than for build).

Reply:
Thanks a lot! This seems much better. I will try this.